helpme/helpme.py

```diff
@@ -451,7 +451,7 @@ def give_general_advice(plugin):
     if len(funds) == 0:
         r += "INCOMPLETE: No bitcoins yet.  Try 'helpme funds'"
     else:
-        funds = Millisatoshi(sum([funds['amount_msat'] for f in funds
+        funds = Millisatoshi(1000 * sum([f['value'] for f in funds
```

Comment:
Would this work if we were to define `__radd__` to the `Millisatoshi` (and `Satoshi`) type? That should add support for these kind of numeric operations :-)

Reply:
Yes, but we can stick with this for backwards compatiblity. I can make an additional PR on the on contrib/pylightning...

Reply:
That'd be very much appreciated, thanks ^^

Reply:
okay. see https://github.com/ElementsProject/lightning/pull/3022